src/prettier.ts

```diff
@@ -0,0 +1,52 @@
+import { join } from "jsr:@std/path";
+import type { DenoConfigType, GlobalConfigType } from "./types.ts";
+
+// TODO:: need to add all supported types conversion 🚨
+// For more help: https://docs.deno.com/runtime/fundamentals/linting_and_formatting
+const configMap: Record<string, string> = {
+  tabWidth: "indentWidth",
+  semi: "semiColons",
+};
+
+const mapRules = (options: GlobalConfigType): GlobalConfigType => {
+  const resOptions: GlobalConfigType = {};
+
+  for (const item in options) {
+    if (Object.hasOwn(configMap, item)) {
+      resOptions[configMap[item]] = options[item];
+    } else {
+      resOptions[item] = options[item];
+    }
+  }
+
+  return resOptions;
+};
+export async function migratePrettierScripts({
+  workingDirectory,
+  existingDenoConfig,
+}: {
+  workingDirectory: string;
+  existingDenoConfig: DenoConfigType;
+}) {
+  try {
+    const configFilePath = join(workingDirectory, ".prettierrc");
```

Comment:
We need to support the rest of the configurations; I hope you can create one more PR to make thing even better

https://github.com/HazemKhaled/deno-migrate/pull/3/files#diff-9ca7a56e2b3e092525c7deb971c434878c1c9f30142d4eb35f937f18ba54f961R12